main.rb

```diff
@@ -1,4 +1,60 @@
 require_relative 'app'
 
-app = App.new
-app.run
+class Main
+  def initialize()
+    @app = App.new
+    @hash = { 1 =>
+    proc do
+      @app.list_books
+    end, 2 =>
+    proc do
+      @app.list_people
+    end, 3 =>
+    proc do
+      @app.create_person
+    end, 4 =>
+    proc do
+      @app.create_book
+    end, 5 =>
+    proc do
+      @app.create_rental
+    end, 6 =>
+    proc do
+      @app.list_rentals
+    end, 7 => end_hash }
+  end
+
+  def display_menu
+    puts 'Welcome to school library App'
+    puts 'Please choose an option to enter in a number'
+    puts '1. List all books'
+    puts '2. List all people'
```

Comment:
- [ ] Kindly remove the second display that comes after the needed person and books are listed. This is merely a refactor, so the app UI is still expected to match the video shared in an earlier project.

  ![ieokd](https://user-images.githubusercontent.com/36542986/226598817-18102b3d-7de2-4409-8da4-71ddf85819cd.jpg)
  ![njs](https://user-images.githubusercontent.com/36542986/226598810-9445a4c8-dc97-4839-841c-5a7f278b9de4.jpg)